src/coreclr/hosts/corerun/corerun.cpp

```diff
@@ -33,7 +33,7 @@ static const wchar_t *coreCLRDll = W("CoreCLR.dll");
 static const wchar_t *coreCLRInstallDirectory = W("%windir%\\system32\\");
 
 // Encapsulates the environment that CoreCLR will run in, including the TPALIST
-class HostEnvironment 
+class HostEnvironment
```

Comment:
Merge conflict?

Reply:
This file and `Directory.Build.targets` are part of local working infrastructure which slipped into commit. Removing.